### ISSUE TITLE
Confusing the use of FlexLine's top (mTop/getTop())

### ISSUE THREAD

**Issue by eneim** [google/flexbox-layout#244]
In my case, FlexLine top is the same for all lines. In fact, it is understandable as the way it was set is as follow:

```
mTop = Math.min(mTop, view.getTop() - flexItem.getMarginTop() - topDecoration);
```

So every FlexLine may shares the same Top, which is confused to me (in a Row/Wrap, each line stays beneath other line accept for the one on top, so a line's top should be bigger than the line above it).

Can you enlighten me more about this mTop if possible?

**Comment by thagikura** [comment]
I assume you are using FlexboxLayoutManager, instead of FlexboxLayout.

And I realized there is an issue on calculation of that logic..
Since when the user scrolls, layoutFlexLineMainAxis(Horizontal/Vertical) is called only the new line, the coordinates for existing lines are not updated.

**Comment by eneim** [comment]
@thagikura Ah yes, to confirm your point, it is about FlexboxLayoutManager. Thanks for the response.

**Comment by thagikura** [comment]
Fixed by #316. Removed the set of methods of retrieving the positions of a flex line.
